examples.js

```diff
@@ -0,0 +1,107 @@
+function examples() {
+
+  setTimeout(blastOff, 86400000);
+
+
+  const locations = ["Austin", "New York", "San Francisco"];
+  locations.forEach(l => {
+    doStuff();
+    doSomeOtherStuff();
+    dispatch(l);
+  });
```

Comment:
Considera ser explícito con la variable dentro de la función, ayuda a tener contexto. 
```suggestion
  const locations = ["Austin", "New York", "San Francisco"];
  locations.forEach(location => {
    doStuff();
    doSomeOtherStuff();
    dispatch(l);
  });
```